src/data/polynomial/degree/definitions.lean

```diff
@@ -169,6 +169,13 @@ lemma nat_degree_le_nat_degree [semiring S] {q : S[X]} (hpq : p.degree ≤ q.deg
   p.nat_degree ≤ q.nat_degree :=
 with_bot.gi_unbot'_bot.gc.monotone_l hpq
 
+lemma nat_degree_lt_nat_degree {p q : R[X]} (hp : p ≠ 0) (hpq : p.degree < q.degree) :
+  p.nat_degree < q.nat_degree :=
+begin
+  by_cases hq : q = 0, { rw [hq, degree_zero] at hpq, have := not_lt_bot hpq, contradiction },
```

Comment:
You don't need `degree_zero`:
```suggestion
  by_cases hq : q = 0, { rw [hq] at hpq, exact (not_lt_bot hpq).elim },
```

Reply:
Thanks! I moved the lemma without golfing it; definitely a good opportunity to golf it.